test/test_l2_projection.jl

```diff
@@ -1,21 +1,17 @@
 # Tests a L2-projection of integration point values (to nodal values),
 # determined from the function y = 1 + x[1]^2 + (2x[2])^2
-function test_projection(order, refshape)
-    element = refshape == RefQuadrilateral ? Quadrilateral : Triangle
-    grid = generate_grid(element, (1, 1), Vec((0.0, 0.0)), Vec((1.0, 1.0)))
 
+function test_projection(order, elementtype)
+    refshape = getrefshape(elementtype)
+    dim = Ferrite.getrefdim(refshape)
+    grid = single_element_grid(elementtype)
```

Comment:
Why did you change this to a single element grid? Previously we had triangles/tetrahedrons filled a square/cube.